server/src/util/jest-db.ts

```diff
@@ -27,7 +27,7 @@ export enum ModelName {
  * Connect to a in-memory database
  */
 export const createTestDatabase = async (): Promise<Sequelize> => {
-  const sequelize = new Sequelize('sqlite::memory:', { logging: false })
+  const sequelize = new Sequelize('sqlite::memory:', { logging: true })
```

Comment:
You forgot to switch logging back off...

Reply:
On it